spec/unit/provider/service/windows_spec.rb

```diff
@@ -511,6 +511,31 @@
         double("StatusStruct", current_state: "running"))
     end
 
+    context "run_as_user user is specified" do
+      let(:run_as_user) { provider.new_resource.class.properties[:run_as_user].default }
```

Comment:
@lamont-granquist Tied back to your original question...

> if the user specifying "LocalSystem" is different than the default of "LocalSystem" that seems, at any rate, odd and worthy of some commented documentation if that's the way things are...

I intentionally setup the test like this so that we would explicitly differentiate between the default being explicitly set and the default itself.